ruled_labels/specs_polkadot.yaml

```
@@ -0,0 +1,250 @@
+    description: "Pull request requires no code review (e.g., a sub-repository hash update)."
+    color: C6C7F8
+  - name: A3-in_progress
+    description: Pull request is in progress. No review needed at this stage.
+    color: DBC6F8
+  - name: A4-companion
+    description: A PR that should be considered alongside another (usually more comprehensive and detailed) PR.
+    color: EEF8C0
+  - name: A5-stale
+    description: Pull request did not receive any updates in a long time. No review needed at this stage. Close it.
+    color: F2C5F8
+  - name: A6-backport
+    description: Pull request is already reviewed well in another branch.
+    color: F8C5E7
+  - name: B0-silent
+    description: Changes should not be mentioned in any release notes
+    color: f9f0f4
+  - name: B1-note_worthy
+    description: Changes should be noted in the release notes
+    color: f9f0f4
+  - name: C1-low
+    description: PR touches the given topic and has a low impact on builders.
+    color: eeeeee
+  - name: C3-medium
+    description: PR touches the given topic and has a medium impact on builders.
+    color: eebb00
+  - name: C5-high
+    description: PR touches the given topic and has a high impact on builders.
+    color: D4C5F9
+  - name: C7-critical
+    description: PR touches the given topic and has a critical impact on builders.
+    color: E99695
+  - name: D1-audited 👍
+    description: PR contains changes to fund-managing logic that has been properly reviewed and externally audited.
+    color: "006600"
+  - name: D2-notlive 💤
+    description: PR contains changes in a runtime directory that is not deployed to a chain that requires an audit.
+    color: 0052CC
+  - name: D3-trivial 🧸
+    description: PR contains trivial changes in a runtime directory that do not require an audit.
+    color: FBCA04
+  - name: D5-nicetohaveaudit ⚠️
+    description: PR contains trivial changes to logic that should be properly reviewed.
+    color: aa6600
+  - name: D9-needsaudit 👮
+    description: PR contains changes to fund-managing logic that should be properly reviewed and externally audited.
+    color: "880000"
+  - name: E0-runtime_migration
+    description: PR introduces code that might require downstream chains to run a runtime upgrade.
+    color: "221133"
+  - name: E1-database_migration
+    description: PR introduces code that does a one-way migration of the database.
+    color: "112233"
+  - name: E2-dependencies
+    description: Pull requests that update a dependency file.
+    color: "222211"
+  - name: E3-host_functions
+    description: PR adds new host functions which requires a node release before a runtime upgrade.
+    color: f9f2f2
+  - name: E4-node_first_update
+    description: This is a runtime change that will require all nodes to be update BEFORE the runtime upgrade.
+    color: e8d7d7
+  - name: F0-breaks_everything
+    description: "This change breaks the underlying networking, sync or related and thus will cause a fork."
```

```suggestion
    description: This change breaks the underlying networking, sync or related and thus will cause a fork.
```
There are no `"` on the other descriptions.